src/components/plugins/PluginIndex.vue

```diff
@@ -17,17 +17,17 @@
             </div>
         </template>
         <template v-else>
-            <div class="d-flex flex-column elements-section" v-for="(elements, elementType) in elementsByType" :key="elementType">
+            <div class="d-flex flex-column elements-section" v-for="(elements, elementType) in groupedByAction" :key="elementType">
                 <h4 :id="`section-${slugify(elementType)}`" class="text-capitalize">
                     {{ elementType }}
                 </h4>
                 <div class="d-flex flex-column">
                     <RowLink
-                        v-for="element in elements"
+                        v-for="element in Object.keys(elements)"
```

Comment:
[vue allows for looping on objects](https://vuejs.org/guide/essentials/list#v-for-with-an-object)

```suggestion
                        v-for="(_, element) of elements"
```